app/gateway/2.7.x/developer-portal/using-the-editor.md

```diff
@@ -19,23 +17,21 @@ Kong Manager offers a robust file editor for editing the template files of the D
 
 From the **Kong Manager** dashboard of your **Workspace**, click **Editor** under **Dev Portal** in the sidebar.
 
-![Launch Editor Mode](https://doc-assets.konghq.com/1.3/dev-portal/editor/devportal-editor-sidebar-button.png)
-
+![Dev Portal comes with a built-in editor. Launch it by clicking on Editor.](/assets/images/docs/dev-portal/editor-dev-portal.png)
 
 This will launch the **Editor Mode**:
 
 ![Editor Mode](https://doc-assets.konghq.com/1.3/dev-portal/editor/devportal-editor-mode-launch.png)
```

Comment:
This screenshot (and the other two screenshots in this file hosted at `https://doc-assets.konghq.com`) are also outdated. Notice the "Kong Enterprise" logo in the navigation, which should now be "Kong Manager":

<img width="456" alt="Screen Shot 2022-02-09 at 11 27 17 AM" src="https://user-images.githubusercontent.com/54370747/153275682-117fbcc8-6454-4998-a192-1309f78985f4.png">

One of the screenshots even has the entirety of the browser address bar in it, which we definitely don't want:

<img width="806" alt="Screen Shot 2022-02-09 at 11 28 14 AM" src="https://user-images.githubusercontent.com/54370747/153275810-0d66a880-dcdf-441b-ab42-098331a1dc41.png">

I recommend either:
* Cropping the navbar out of each old screenshot, or
* Delete all screenshots of the editor and update only the numbered screenshot.
Either way, you won't be able to crop the numbered screenshot directly, as one of the numbers overlaps with the navbar.